### PR TITLE
ICS4: Minor Upgrade Requests from Implementation Team

### PR DESCRIPTION
**Pull request by AdityaSripal**

### CODE REVIEW

**Comment by AdityaSripal** [comment]
This doesn't feel like it needs a changelog. Let me know if I should add any entries

File: spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md [hunk 1]
```diff
@@ -424,30 +442,13 @@ function chanUpgradeTry(
...
-        restoreChannel()
+        restoreChannel(portIdentifier, channelIdentifier)
         return
     }
 
-    // both channel ends must be mutually compatible.
-    // this function has been left unspecified since it will depend on the specific structure of the new channel.
-    // It is the responsibility of implementations to make sure that the proposed new channels
-    // on either side are correctly constructed according to the new version selected.
-    if !IsCompatible(counterpartyChannel, proposedUpgradeChannel) {
```
Comment (mpoke) [inline spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md:447]: If I understand correctly, we replaced this with 
```
abortTransactionUnless(counterpartyChannel.ordering != proposedUpgradeChannel.ordering)
```
 to only focus on ordering, right? We should do the same in `chanUpgradeAck`.

**Review by mpoke — APPROVED** [review]
Nice work.

File: spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md (hunk 1 above)
Comment (AdityaSripal) [reply spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md:449]: Good catch 👍

**Pull request merged**